### PR TITLE
Bug 1740121: Make service-ca have guaranteed scheduling

### PR DESCRIPTION
**Pull request by enj**
Signed-off-by: Monis Khan <mkhan@redhat.com>

### CODE REVIEW

File: hack/verify-generated-bindata.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 function cleanup() {
     return_code=$?
     rm -rf "${TMP_GENERATED_BOOTSTRAP_DIR}"
-    os::test::junit::generate_report
+    # os::test::junit::generate_report
```
Comment (sallyom) [inline hack/verify-generated-bindata.sh]: uncomment, right?

Comment (enj) [reply hack/verify-generated-bindata.sh]: `os::test::junit::generate_report` does not exist.  We need to delete all of this and move to the lib-go `make` stuff anyway.

Comment (sallyom) [reply hack/verify-generated-bindata.sh]: oh i see, nm, we need to update CI/update to library-go shell scripts, etc.

**Pull request merged**